### PR TITLE
Fix test version package bump

### PR DESCRIPTION
**Pull request by andrewlock**
## Summary of changes

- Fix the Version Bump action that has been broken for a while
- Run the version bump action, to test latest packages

## Reason for change

#3146 added support for `System.Diagnostics.Process` which broke the version bump action, but we didn't notice. This fixes it, and bumps to the latest verions

## Implementation details

Adds `System.Diagnostics.Process` to the known list of integrations

## Test coverage

Let's see how it goes...

## Other details

Possible additional actions:
- Add new action which checks for this in the PR. This is probably the best option.
- Don't _require_ the instrumented assembly to be documented. Downside is that we could miss adding checks for packages that we want to.

### CODE REVIEW

File: tracer/dependabot/Datadog.Dependabot.Integrations.csproj [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
     <!-- Integration: GraphQL -->
     <!-- Assembly: GraphQL -->
-    <!-- Latest package https://www.nuget.org/packages/GraphQL/7.0.2 -->
+    <!-- Latest package https://www.nuget.org/packages/GraphQL/7.1.1 -->
     <PackageReference Include="GraphQL" Version="5.3.3" />
```
Comment (andrewlock) [inline tracer/dependabot/Datadog.Dependabot.Integrations.csproj:57]: We don't yet support 7.x as requires some work on the test app

**Comment by andrewlock** [comment]
## Benchmarks Report :snail:

Benchmarks for #3291 compared to [master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c): 
* All benchmarks have **the same** speed
* All benchmarks have **the same** allocations


The following thresholds were used for comparing the benchmark speeds:
* Mann–Whitney U test with statistical test for significance of **5%**
* Only results indicating a difference greater than **10%** and **0.3 ns** are considered.

Allocation changes below **0.5%** are ignored.


### Benchmark details


<details><summary><strong>Benchmarks.Trace.AgentWriterBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`WriteAndFlushEnrichedTraces`|net472|726μs|284ns|1.06μs|0.357| 0|0|3.18&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|476μs|163ns|609ns|0| 0|0|2.58&nbsp;KB|
|#3291|`WriteAndFlushEnrichedTraces`|net472|710μs|523ns|2.02μs|0.357| 0|0|3.18&nbsp;KB|
|#3291|`WriteAndFlushEnrichedTraces`|netcoreapp3.1|459μs|118ns|442ns|0| 0|0|2.58&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AppSecBodyBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`AllCycleSimpleBody`|net472|196ns|0.0835ns|0.313ns|0.0726| 0|0|457&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`AllCycleSimpleBody`|netcoreapp3.1|257ns|0.124ns|0.481ns|0.00629| 0|0|456&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`AllCycleMoreComplexBody`|net472|192ns|0.0766ns|0.276ns|0.0688| 0|0|433&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`AllCycleMoreComplexBody`|netcoreapp3.1|263ns|1.35ns|6.61ns|0.00591| 0|0|432&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`BodyExtractorSimpleBody`|net472|274ns|0.115ns|0.43ns|0.0573| 0|0|361&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`BodyExtractorSimpleBody`|netcoreapp3.1|235ns|0.124ns|0.48ns|0.00373| 0|0|272&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`BodyExtractorMoreComplexBody`|net472|15.8μs|13.8ns|51.6ns|1.21| 0.0159|0|7.62&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`BodyExtractorMoreComplexBody`|netcoreapp3.1|13.3μs|5.73ns|22.2ns|0.0928| 0|0|6.75&nbsp;KB|
|#3291|`AllCycleSimpleBody`|net472|196ns|0.0439ns|0.158ns|0.0726| 0|0|457&nbsp;B|
|#3291|`AllCycleSimpleBody`|netcoreapp3.1|258ns|0.208ns|0.805ns|0.00632| 0|0|456&nbsp;B|
|#3291|`AllCycleMoreComplexBody`|net472|193ns|0.117ns|0.438ns|0.0688| 0|0|433&nbsp;B|
|#3291|`AllCycleMoreComplexBody`|netcoreapp3.1|251ns|0.216ns|0.807ns|0.00592| 0|0|432&nbsp;B|
|#3291|`BodyExtractorSimpleBody`|net472|273ns|0.0983ns|0.368ns|0.0573| 0|0|361&nbsp;B|
|#3291|`BodyExtractorSimpleBody`|netcoreapp3.1|244ns|0.109ns|0.393ns|0.00371| 0|0|272&nbsp;B|
|#3291|`BodyExtractorMoreComplexBody`|net472|16.5μs|10.1ns|39.2ns|1.21| 0.0164|0|7.62&nbsp;KB|
|#3291|`BodyExtractorMoreComplexBody`|netcoreapp3.1|12.9μs|5.37ns|20.1ns|0.09| 0|0|6.75&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.AspNetCoreBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`SendRequest`|netcoreapp3.1|179μs|150ns|581ns|0.268| 0|0|20.68&nbsp;KB|
|#3291|`SendRequest`|net472|0ns|0ns|0ns|0| 0|0|0&nbsp;b|
|#3291|`SendRequest`|netcoreapp3.1|179μs|312ns|1.21μs|0.179| 0|0|20.68&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.DbCommandBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`ExecuteNonQuery`|net472|1.82μs|1.05ns|3.92ns|0.155| 0.000909|0|979&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`ExecuteNonQuery`|netcoreapp3.1|1.42μs|0.723ns|2.8ns|0.0128| 0|0|968&nbsp;B|
|#3291|`ExecuteNonQuery`|net472|1.94μs|0.749ns|2.8ns|0.155| 0.000968|0|979&nbsp;B|
|#3291|`ExecuteNonQuery`|netcoreapp3.1|1.46μs|0.679ns|2.54ns|0.0131| 0|0|968&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.ElasticsearchBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`CallElasticsearch`|net472|2.5μs|0.655ns|2.54ns|0.188| 0|0|1.19&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`CallElasticsearch`|netcoreapp3.1|1.54μs|0.519ns|1.87ns|0.0147| 0|0|1.13&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`CallElasticsearchAsync`|net472|2.48μs|0.623ns|2.41ns|0.21| 0|0|1.32&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`CallElasticsearchAsync`|netcoreapp3.1|1.58μs|0.45ns|1.69ns|0.0166| 0|0|1.25&nbsp;KB|
|#3291|`CallElasticsearch`|net472|2.46μs|0.937ns|3.63ns|0.188| 0|0|1.19&nbsp;KB|
|#3291|`CallElasticsearch`|netcoreapp3.1|1.61μs|1.39ns|5.2ns|0.0153| 0|0|1.13&nbsp;KB|
|#3291|`CallElasticsearchAsync`|net472|2.71μs|0.959ns|3.59ns|0.209| 0|0|1.32&nbsp;KB|
|#3291|`CallElasticsearchAsync`|netcoreapp3.1|1.62μs|0.516ns|2ns|0.0171| 0|0|1.25&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.GraphQLBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`ExecuteAsync`|net472|2.77μs|1.36ns|4.9ns|0.23| 0|0|1.45&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`ExecuteAsync`|netcoreapp3.1|1.81μs|0.681ns|2.64ns|0.0181| 0|0|1.38&nbsp;KB|
|#3291|`ExecuteAsync`|net472|2.81μs|1.36ns|4.89ns|0.23| 0|0|1.45&nbsp;KB|
|#3291|`ExecuteAsync`|netcoreapp3.1|1.75μs|0.435ns|1.63ns|0.0184| 0|0|1.38&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.HttpClientBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`SendAsync`|net472|5.93μs|1.99ns|7.17ns|0.445| 0|0|2.8&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`SendAsync`|netcoreapp3.1|3.77μs|1.81ns|6.79ns|0.0359| 0|0|2.63&nbsp;KB|
|#3291|`SendAsync`|net472|5.84μs|1.47ns|5.5ns|0.444| 0|0|2.8&nbsp;KB|
|#3291|`SendAsync`|netcoreapp3.1|3.74μs|1.76ns|6.59ns|0.0356| 0|0|2.63&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.ILoggerBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|net472|3.28μs|2.28ns|8.84ns|0.297| 0|0|1.88&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|netcoreapp3.1|2.63μs|1.76ns|6.36ns|0.0261| 0|0|1.91&nbsp;KB|
|#3291|`EnrichedLog`|net472|3.23μs|3.07ns|11.9ns|0.297| 0|0|1.88&nbsp;KB|
|#3291|`EnrichedLog`|netcoreapp3.1|2.48μs|9.44ns|36.6ns|0.0256| 0|0|1.91&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.Log4netBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|net472|151μs|171ns|661ns|0.749| 0.225|0|4.72&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|netcoreapp3.1|117μs|326ns|1.26μs|0.0585| 0|0|4.55&nbsp;KB|
|#3291|`EnrichedLog`|net472|154μs|164ns|634ns|0.687| 0.229|0|4.72&nbsp;KB|
|#3291|`EnrichedLog`|netcoreapp3.1|118μs|293ns|1.13μs|0.0593| 0|0|4.55&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.NLogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|net472|5.84μs|2.04ns|7.92ns|0.579| 0.00291|0|3.65&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|netcoreapp3.1|4.5μs|1.75ns|6.32ns|0.0539| 0|0|3.98&nbsp;KB|
|#3291|`EnrichedLog`|net472|5.85μs|1.84ns|7.12ns|0.58| 0.00294|0|3.65&nbsp;KB|
|#3291|`EnrichedLog`|netcoreapp3.1|4.49μs|1.68ns|6.53ns|0.0539| 0|0|3.98&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.RedisBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`SendReceive`|net472|2.2μs|3.59ns|13.4ns|0.222| 0|0|1.4&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`SendReceive`|netcoreapp3.1|1.85μs|1.41ns|5.29ns|0.0187| 0|0|1.35&nbsp;KB|
|#3291|`SendReceive`|net472|2.27μs|3.52ns|12.7ns|0.223| 0|0|1.4&nbsp;KB|
|#3291|`SendReceive`|netcoreapp3.1|1.75μs|2.34ns|9.06ns|0.0183| 0|0|1.35&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SerilogBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|net472|5.02μs|2.22ns|8.6ns|0.365| 0|0|2.3&nbsp;KB|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`EnrichedLog`|netcoreapp3.1|4.23μs|1.31ns|5.06ns|0.0254| 0|0|1.86&nbsp;KB|
|#3291|`EnrichedLog`|net472|5.09μs|2.24ns|8.38ns|0.363| 0|0|2.3&nbsp;KB|
|#3291|`EnrichedLog`|netcoreapp3.1|4.2μs|1.53ns|5.92ns|0.0252| 0|0|1.86&nbsp;KB|
</details>


<details><summary><strong>Benchmarks.Trace.SpanBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`StartFinishSpan`|net472|1.16μs|0.193ns|0.72ns|0.134| 0|0|842&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`StartFinishSpan`|netcoreapp3.1|929ns|1.15ns|4.29ns|0.0106| 0|0|792&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`StartFinishScope`|net472|1.43μs|0.281ns|1.09ns|0.146| 0|0|923&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`StartFinishScope`|netcoreapp3.1|1.09μs|0.579ns|2.09ns|0.0126| 0|0|912&nbsp;B|
|#3291|`StartFinishSpan`|net472|1.21μs|0.704ns|2.73ns|0.134| 0|0|842&nbsp;B|
|#3291|`StartFinishSpan`|netcoreapp3.1|944ns|0.303ns|1.17ns|0.0109| 0|0|792&nbsp;B|
|#3291|`StartFinishScope`|net472|1.44μs|0.464ns|1.74ns|0.147| 0|0|923&nbsp;B|
|#3291|`StartFinishScope`|netcoreapp3.1|1.11μs|0.401ns|1.5ns|0.0122| 0|0|912&nbsp;B|
</details>


<details><summary><strong>Benchmarks.Trace.TraceAnnotationsBenchmark</strong> - Same speed :heavy_check_mark: Same allocations :heavy_check_mark:

</summary>

### Raw results


|  Branch |                      Method |     Toolchain |     Mean | StdError |  StdDev |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|---------|---------------------------- |-------------- |---------:|---------:|--------:|-------:|------:|------:|----------:|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`RunOnMethodBegin`|net472|1.47μs|0.499ns|1.87ns|0.146| 0|0|923&nbsp;B|
|[master](https://github.com/DataDog/dd-trace-dotnet/tree/a8723af695a54c6fd62c4d21e5cc91c8a527d04c)|`RunOnMethodBegin`|netcoreapp3.1|1.2μs|0.459ns|1.66ns|0.0125| 0|0|912&nbsp;B|
|#3291|`RunOnMethodBegin`|net472|1.51μs|0.447ns|1.67ns|0.147| 0|0|923&nbsp;B|
|#3291|`RunOnMethodBegin`|netcoreapp3.1|1.19μs|0.263ns|0.947ns|0.0121| 0|0|912&nbsp;B|
</details>

**Review by pierotibou — APPROVED** [review]
Thanks a lot for finding and fixing this. 
Have you looked into having some monitoring around this so that we get notified?

**Comment by andrewlock** [comment]
> Have you looked into having some monitoring around this so that we get notified?

Nope! We could create a github action that monitors github actions, to make sure the github action passes. But then what happens if the monitoring github action fails 🤔 🐢🐢🐢🐢🐢🐢

**Comment by andrewlock** [comment]
## Code Coverage Report :bar_chart:

:heavy_check_mark: Merging #3291 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/00bc1c9abda49ea4aeeba329431af74d8aeaea81) will **not change** line coverage
:heavy_check_mark: Merging #3291 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/00bc1c9abda49ea4aeeba329431af74d8aeaea81) will **not change** branch coverage
:heavy_check_mark: Merging #3291 into [master](https://github.com/DataDog/dd-trace-dotnet/tree/00bc1c9abda49ea4aeeba329431af74d8aeaea81) will **not change** complexity


|           | [master](https://github.com/DataDog/dd-trace-dotnet/tree/00bc1c9abda49ea4aeeba329431af74d8aeaea81) | #3291       | Change   | 
|:----------|:-----------:|:-----------:|:--------:|
| Lines     | `18342` / `25593` | `18429` / `25593` |          |
| Lines %   | `72%`      | `72%`      |  `0%` :heavy_check_mark:  |
| Branches  | `10626` / `15583` | `10646` / `15583` |          |
| Branches %| `68%`      | `68%`      |  `0%` :heavy_check_mark:  |
| Complexity|   `16844`      | `16844`        |  `0`  :heavy_check_mark:    |

View the full report for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=113951&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/113951/artifacts?artifactName=Code%20Coverage%20Report_113951&api-version=6.0&%24format=zip)  
* [HTML report for this PR #3291](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=113947&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/113947/artifacts?artifactName=Code%20Coverage%20Report_113947&api-version=6.0&%24format=zip)

### Datadog.Trace Breakdown :heavy_check_mark:

|        | [master](https://github.com/DataDog/dd-trace-dotnet/tree/00bc1c9abda49ea4aeeba329431af74d8aeaea81) | #3291       | Change   | 
|:-------|:-----------:|:-----------:|:--------:|
| Lines %| `72%`      | `72%`       |  `0%` :heavy_check_mark:  |
| Branches %| `68%`      | `68%`       |  `0%` :heavy_check_mark:  |
| Complexity| `16844`      | `16844`       |  `0` :heavy_check_mark:  |

The following classes have significant coverage changes.

| File    | Line coverage change | Branch coverage change | Complexity change |
|:--------|:--------------------:|:----------:|:--------:|
| [Datadog.Trace.Ci.GitInfo](https://github.com/DataDog/dd-trace-dotnet/tree/654cc020f6f72109ed1f37d0eee43b816772f983/tracer/src/Datadog.Trace/Ci/GitInfo.cs) | `38%` :heavy_check_mark: | `30%` :heavy_check_mark:   | `0` :heavy_check_mark: |


View the full reports for further details:

* [HTML report for master](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=113951&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/113951/artifacts?artifactName=Code%20Coverage%20Report_113951&api-version=6.0&%24format=zip)  
* [HTML report for this PR #3291](https://dev.azure.com/datadoghq/dd-trace-dotnet/_build/results?buildId=113947&view=codecoverage-tab) | [Source file](https://dev.azure.com/datadoghq/a51c4863-3eb4-4c5d-878a-58b41a049e4e/_apis/build/builds/113947/artifacts?artifactName=Code%20Coverage%20Report_113947&api-version=6.0&%24format=zip)

**Review by zacharycmontoya — APPROVED** [review]

**Comment by pierotibou** [comment]
It amazes me than when I say monitor, you don't think Datadog! :) 
Maybe the failures are visible in CI monitoring. I'll check tmr

File: tracer/build/_build/Honeypot/IntegrationGroups.cs [hunk 2]
```diff
@@ -23,6 +23,7 @@ static IntegrationMap()
             NugetPackages.Add("netstandard", new string[] { });
             NugetPackages.Add("System.Messaging", new string[] { });
             NugetPackages.Add("System", new string[] { });
+            NugetPackages.Add("System.Diagnostics.Process", new string[] { });
```
Comment (lucaspimentel) [inline tracer/build/_build/Honeypot/IntegrationGroups.cs:26]: This syntax 😖

**Pull request merged**